js/main.js

```diff
@@ -240,43 +242,89 @@ const createPinsNodeFragment = (pinsArr) => {
   return fragment;
 };
 
-// const createСardFragment = (cardObj) => {
-//   const fragment = document.createDocumentFragment();
-//   fragment.appendChild(createCard(cardObj));
-//   return fragment;
-// };
+const createСardFragment = (cardObj) => {
+  const fragment = document.createDocumentFragment();
+  fragment.appendChild(createCard(cardObj));
+  return fragment;
+};
+
+const pinsDataArray = createDataArray(PINS_AMOUNT);
 
 const initPinsScreen = () => {
-  const pinsDataArray = createDataArray(PINS_AMOUNT);
   const pinsNodesFragment = createPinsNodeFragment(pinsDataArray);
-  // const cardNodesFragment = createСardFragment(pinsDataArray[0]);
-
-  // mapNode.insertBefore(cardNodesFragment, mapFiltersNode);
   mapPinsNode.appendChild(pinsNodesFragment);
+};
 
+const validateCheckInInput = () => {
+  if (formNode.timein.value) {
+    formNode.timeout.value = formNode.timein.value;
+  }
 };
 
-const validateRoomsInput = () => {
+const validateCheckOutInput = () => {
+  if (formNode.timeout.value) {
+    formNode.timein.value = formNode.timeout.value;
+  }
+};
 
-  const a = !ROOMS_FOR_GUESTS_MAP[formNode.rooms.value].includes(formNode.capacity.value) ? formNode.capacity.setCustomValidity(`Вы не можете выбрать данное количество гостей`) : formNode.capacity.setCustomValidity(``);
+const validateRoomsInput = () => {
+  formNode.capacity.setCustomValidity(ROOMS_FOR_GUESTS_MAP[formNode.rooms.value].includes(formNode.capacity.value) ? `` : `Вы не можете выбрать данное количество гостей`);
   formNode.capacity.reportValidity();
-  return a;
+};
+
+const validatePriceInput = () => {
+  if (formNode.price.value > MAX_PRICE) {
+    formNode.price.setCustomValidity(`Вы превысили масимальную цену за номер на ${new Intl.NumberFormat(`ru-RU`).format(formNode.price.value - MAX_PRICE)} руб.`);
+  } else {
+    formNode.price.setCustomValidity(formNode.price.value >= MIN_PRICE[formNode.type.value] ? `` : `Минимальцая цена за номер ${new Intl.NumberFormat(`ru-RU`).format(MIN_PRICE[formNode.type.value])} руб.`);
+  }
+
+  formNode.price.reportValidity();
+};
+
+const validateTitleInput = () => {
+  const valueLength = formNode.title.value.length;
+
+  if (valueLength < TitleLength.MIN) {
+    formNode.title.setCustomValidity(`Ещё ${TitleLength.MIN - valueLength} симв.`);
+  } else if (valueLength > TitleLength.MAX) {
+    formNode.title.setCustomValidity(`Удалите лишние ${valueLength - TitleLength.MAX} симв.`);
+  } else {
+    formNode.title.setCustomValidity(``);
+  }
+  formNode.title.reportValidity();
+};
+
+const validateForm = () => {
+  validateRoomsInput();
+  validateTitleInput();
+  validatePriceInput();
 };
```

Comment:
Можно упростить себе жизнь и реализовать это через `minlength` и `maxlength` в верстке)
```suggestion
const validateForm = () => {
  validateRoomsInput();
  validatePriceInput();
};
```